definitions/private-law/json/CaseEventToComplexTypes/ServiceOfApplication/ConfirmRecipients/CaseEventToComplexTypes.json

```diff
@@ -45,7 +45,7 @@
     "ID": "ConfirmRecipients",
     "CaseEventID": "serviceOfApplication",
     "CaseFieldID": "confirmRecipients",
-    "ListElementCode": "otherPeopleList",
+    "ListElementCode": "respondentsList",
```

Comment:
@BharadwajSairam , please verify this file change. seems to be a bit misleading. we are missing out changes from master here.